docs/EVM-compatible.md

```diff
@@ -39,6 +39,12 @@ When you send an Ethereum transaction, the transaction is converted to Godwoken
 | GASLIMIT | `block.gaslimit` | 12,500,000 | current block's gas limit |
 | DIFFICULTY | `block.difficulty` | 2,500,000,000,000,000 | current block's difficulty |
 
+### Restriction of memory usage
+
+Polyjuice runs EVM on the ckb-vm. Although there is no limit on memory usage of EVM(well, EVM does have a limit on stack depth which is 1024), ckb-vm can only use 4MB memory at most for now.
+In ckb-vm, 3MB for code space and 1MB for stack space. See more details in [here](https://github.com/nervosnetwork/riscv-newlib/blob/00c6ae3c481bc62b4ac016b3e86c508cdf2e68d2/libgloss/riscv/sys_sbrk.c#L38-L56). 
+It means that, for some contracts, which consume a lot of memory or have deep call stacks, those contracts could be not compatible on the ckb-vm.
```

Comment:
For some contracts that consume a lot of memory or that have deep call stacks, this may indicate a potential incompatibility on ckb-vm.